src/footbal.js

```diff
@@ -1,9 +1,9 @@
 const axios = require('axios');
-const WORLD_CUP_URL = 'http://api.football-data.org/v1/competitions/467/fixtures';
+const WORLD_CUP_URL = 'http://api.football-data.org/v2/competitions/2267/matches';
```

Comment:
@gusma This is working? Throws me:

{"message":"The resource you are looking for does not exist.","error":404}

Reply:
@diego-sorribas Hey Diego, it does and it doesn't: the structure of the URL is ok (and their documentation states so), but the id is uncertain, because the WC has not been loaded yet as there are no fixtures/schedule still. Perhaps there will be an update to this today, though I seriously doubt it (since football-data.org as a provider seems kind of dormant). That's why also the PR says we probably should consider an alternate API provider if this information isn't properly updated over the next couple of months.

Reply:
This happens when you don't read the PR description, I jumped directly to the code 😂, my bad! Thanks @gusma